src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -234,15 +234,15 @@ def _process_resource_change(
         # Looks up the human name for the model. Defaults to the model name.
         name = RESOURCE_RENAMES.get(model.__name__, model.__name__.lower())
 
-    event = f"{name}.{action}"
-    if event not in VALID_EVENTS:
+    try:
+        event = SentryAppEventType(f"{name}.{action}")
+    except ValueError as e:
```

Comment:
raising the error here will exclude it from the SLOs, but i think it's ok for now

Reply:
I think it's fine, we'll get a sentry error anyway so it's not like there will not be alerting. This spot it a little awk anyway because what would the event type be for the context manager, unless we have a mini slo for if somethings a valid event but I dont think that's worth adding more cardinality for 1 check.